test/link/import_memory.txt

```diff
@@ -0,0 +1,41 @@
+;;; TOOL: run-wasm-link
+;;; FLAGS:
+(module
+  (import "envA" "memory" (memory $0 1))
+)
+(module
+  (import "envB" "memory" (memory $0 1))
+  (func $foo (result i32)
+    (i32.const 41)
+  )
+)
+(;; STDOUT ;;;
+
+linked.wasm:	file format wasm 0x1
+
+Sections:
+
+     Type start=0x0000000a end=0x0000000f (size=0x00000005) count: 1
+   Import start=0x00000015 end=0x00000025 (size=0x00000010) count: 1
+ Function start=0x0000002b end=0x0000002d (size=0x00000002) count: 1
+     Code start=0x0000002f end=0x00000035 (size=0x00000006) count: 1
+   Custom start=0x0000003b end=0x0000004c (size=0x00000011) "name"
+
+Section Details:
+
+Type:
+ - [0] () -> i32
+Import:
+ - envA.memory -> memory
```

Comment:
It seems that wasm-objdump doesn't display enough information here; I'd like to see the new size of the memory import.